src/components/DeploymentsTable.vue

```diff
@@ -5,6 +5,7 @@
         <ResultsCount v-if="selectedDeployments.length == 0" label="Deployment" :count="total" />
         <SelectedCount v-else :count="selectedDeployments.length" />
 
+        <!-- todo: audit [cchoy] -->
```

Comment:
this is kinda tricky along with the other check further down. 

perhaps in practice this is actually _okay_ - if I don't have workspace permissions to all deployments, but I have manage on a specific one via an ACL - maybe it's _okay_ that we don't let those folks delete that one specific one. just... might be non-intuitive. 

otherwise yeah this would take some thinking. Like we'd maybe have to show the checkbox column `if some(deployments.map(d => d.can.delete)` and then in each row, disable the checkbox if `!deployment.can.delete`.

Reply:
nvm while writing down this comment I had a thought / figure-this-out. 

addressed in 04a9adeb28be4ff87e0bbbb4f8a6727161ec9357